plugins/module_utils/_api_data.py

```diff
@@ -132,13 +132,23 @@ def join_path(path):
     ),
     ('interface', 'list'): APIData(
         primary_keys=('name', ),
+        fully_understood=True,
         fields={
             'comment': KeyInfo(can_disable=True, remove_value=''),
             'exclude': KeyInfo(),
             'include': KeyInfo(),
             'name': KeyInfo(),
         },
     ),
+    ('interface', 'list', 'member'): APIData(
+        primary_keys=('interface', 'list', ),
+        fully_understood=True,
+        fields={
+            'comment': KeyInfo(can_disable=True, remove_value=''),
+            'interface': KeyInfo(),
+            'list': KeyInfo(),
```

Comment:
At least on RouterOS 6, there's also a `disabled` key:
```suggestion
            'list': KeyInfo(),
            'disabled': KeyInfo(default=False),
```

Reply:
oh, right. I didn't test it on ROS6. Will add it

Reply:
and actually it is in ROS7 as well.